### DISCUSSION TITLE
Is it possible to automatically edit the index.html file?

### DISCUSSION THREAD

**Discussion by soliblue** [necolas/react-native-web#2284] [Help]
I need to edit the index.html in order to 

1. Update the background colour of <body>, e.g.

```
<body style="background-color: #131219">
...
</body>
```

2. Add a script tag to <head>

```
<head>
  <script>
  ....
  </script>
  ...
</head>
```
Right now, I do it manually after building the files but it would be great if there is an automated solution for this. 

Thank you in advance!

**Comment by soliblue** [comment]
Hi @necolas,

would you be able to support me here please?

**Comment by yuxizhe** [comment]
@AhmedSoli  You can use nextjs  _document.js and _app.js to achieve this 


https://github.com/vercel/next.js/blob/canary/examples/with-react-native-web/pages/_document.js#L30

https://github.com/vercel/next.js/blob/canary/examples/with-react-native-web/pages/_app.js#L7-L9

https://nextjs.org/docs/advanced-features/custom-document

https://nextjs.org/docs/advanced-features/custom-app
